src/collectors/memory/memory.py

```diff
@@ -81,8 +85,8 @@ def collect(self):
                     name = name.rstrip(':')
                     value = int(value)
 
-                    if ((name not in _KEY_MAPPING and
-                         'detailed' not in self.config)):
+                    if (name not in _KEY_MAPPING
+                            and 'detailed' not in self.config):
```

Comment:
So I just went through a whole PEP8 clean up and opted to use the double brackets instead of the offset statements. Is there a preference for this? I personally find this less readable.